### ISSUE TITLE
Action resolution process?

### ISSUE THREAD

**Issue by withinfocus** [rhysd/actionlint#90]
Hello there -- we use an Action -- specifically https://github.com/actions/stale -- and with a recent release we noticed this linter (invoked via https://github.com/github/super-linter) isn't happy about it. Sample workflow config:

```yaml
...

jobs:
  stale:
    name: Staleness
    runs-on: ubuntu-latest
    permissions:
      pull-requests: write

    steps:
      - uses: actions/stale@v4
        with:
          exempt-draft-pr: true

...
```

Note the `@v4` release. Based on https://github.com/actions/stale/tags this is linked to the latest `@v4.1.0` and the Action runs correctly, but this linter doesn't like `exempt-draft-pr` which seems acceptable based on the `action.yml` present in that repository. Furthermore, if we put almost anything in place instead of `@v4` the linter will pass, and we assume that's some kind of resolution problem altogether.

Can there be some documentation expansions about the resolution process of a config? Even though this is a specific case, why is the linter not seeing this Action's latest properties?

**Comment by rhysd** [comment]
Currently actionlint depends on its own data set of popular actions. The data set is updated periodically but has some delays. It was introduced not to cause many network requests so that actionlint can run faster.

I agree that this causes some pain. I'm considering to implement to fetch action metadata via network, but I can't say when it is implemented yet.
